### PR TITLE
MGMT-16452: Change MCE subscription to use the default channel

### PR DESCRIPTION
**Pull request by danmanor**
Change MCE subscription to use the default channel in order to reduce redundant complexity and maintaining configuration of OCP -> MCE releases

## List all the issues related to this PR

- [ ] New Feature 
- [x] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [x] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: internal/operators/mce/mce_operator_test.go [hunk 1]
```diff
@@ -78,19 +78,16 @@ var _ = Describe("MCE Operator", func() {
...
 	})
 	Context("ValidateCluster", func() {
-		mceMinOpenshiftVersion, err := getMinMceOpenshiftVersion(operator.config.OcpMceVersionMap)
-		Expect(err).ToNot((HaveOccurred()))
-
 		table.DescribeTable("validate cluster when ", func(cluster *common.Cluster, expectedResult api.ValidationResult) {
 			res, _ := operator.ValidateCluster(ctx, cluster)
 			Expect(res).Should(Equal(expectedResult))
 		},
 			table.Entry("Openshift version less than minimal",
```
Comment (eifrach) [inline internal/operators/mce/mce_operator_test.go:85]: can we add a test case for pre-release version ? i.e 4.15-rc0

I've seen it fail in other operator - let make sure this is always working

File: internal/operators/mce/mce_operator.go [hunk 2]
```diff
@@ -84,16 +80,15 @@ func (o *operator) ValidateCluster(_ context.Context, cluster *common.Cluster) (
...
-	}
-	minOpenshiftVersionForMce, err = version.NewVersion(*mceMinOpenshiftVersion)
+
+	minOpenshiftVersionForMce, err = version.NewVersion(o.config.MceMinOpenshiftVersion)
+
 	if err != nil {
 		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{err.Error()}}, nil
 	}
+
 	if ocpVersion.LessThan(minOpenshiftVersionForMce) {
```
Comment (eifrach) [inline internal/operators/mce/mce_operator.go:90]:
```suggestion
	if ok, _ := common.BaseVersionLessThan(minOpenshiftVersionForMce.String(), ocpVersion.String()); ok {
```

**Comment by eifrach** [comment]
/hold 

waiting for fix 
https://github.com/danmanor/assisted-service/pull/1

**Comment by danmanor** [comment]
@eifrach For clarity sake lets merge this one and fix the version comparison in another PR

**Pull request merged**